From f498503889b8178f165afa66dc33aa8ad8901371 Mon Sep 17 00:00:00 2001
From: Andrei Dinu <andrei.adrianx.dinu@intel.com>
Date: Fri, 28 Jun 2013 09:38:08 +0300
Subject: [PATCH] removed the g_type_init()

The function is depricated in glib >= 2.35.0 and is
automatically called.

Signed-off-by: Andrei Dinu <andrei.adrianx.dinu@intel.com>
---
 base/openvas_file.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/base/openvas_file.c b/base/openvas_file.c
index 8597198..d110c7f 100644
--- a/base/openvas_file.c
+++ b/base/openvas_file.c
@@ -164,7 +164,6 @@ openvas_file_copy (const gchar *source_file, const gchar *dest_file)
   GFile *sfile, *dfile;
   GError *error;
 
-  g_type_init ();
   sfile = g_file_new_for_path (source_file);
   dfile = g_file_new_for_path (dest_file);
   error = NULL;
@@ -200,7 +199,6 @@ openvas_file_move (const gchar *source_file, const gchar *dest_file)
   GFile *sfile, *dfile;
   GError *error;
 
-  g_type_init ();
   sfile = g_file_new_for_path (source_file);
   dfile = g_file_new_for_path (dest_file);
   error = NULL;
-- 
1.7.9.5

